scripts/upgradeDependencies/parseUpgradedPackages.js

```diff
@@ -1 +1,35 @@
-module.exports = output => [...new Set(output.match(/^.+?(?=:\s{$)/gm))];
+const outputReducer = (accumulator, curr) => {
```

Comment:
If you remove `--jsonUpgraded` from the commands and do
```
console.log(output.split('\n').filter(str => str.includes('→')).join('\n'));
```
You get a really easy to digest
```
@bbc/psammead:  @bbc/psammead-assets  ^2.2.3  →  ^2.2.4
@bbc/psammead-image-placeholder:  @bbc/psammead-assets  ^2.2.3  →  ^2.2.4
```
Could this be easier to parse?

Reply:
Yeah. This is easier to parse

Reply:
This creates a phlegm down the line because we need to parse the text before use. It's much easier to use `JSON.parse`. So I this won't be efficient eventually

Reply:
>This creates a phlegm down the line because we need to parse the text before use

Sorry I dont follow what you mean, could you give an example please?

Reply:
I was talking about the CHANGELOG text. Initially I felt it might be a bit much, but then After a couple of test runs, it actually works out fine, so I've omitted the `jsonUpgraded` flag